### PR TITLE
more direction towards pagination

### PR DESCRIPTION
**Pull request by mmoghaddam385**
Hopefully adding a mention of the "next_page_path" in the `limit` query param doc + an example in the sample response body will help with the confusion around how to paginate requests.

### CODE REVIEW

File: _specs_/rest/paths/reference/vXListTickers.yml [hunk 1]
```diff
@@ -88,7 +88,12 @@ parameters:
     example: asc
   - name: limit
     in: query
-    description: Limit the size of the response, default is 100 and max is 500.
+    description: |
+      Limit the size of the response, default is 100 and max is 500.
+
+      If your query returns more than the max limit and you want to retrieve the next page of results,
+      see the `next_page_path` response attribute.
```
Comment (mmoghaddam385) [inline _specs_/rest/paths/reference/vXListTickers.yml:95]: Is it possible to link to specific response attributes in our docs UI? If you could click `next_page_path` and have it scroll you down to that response attribute that would be cool

Comment (jbonzo) [reply _specs_/rest/paths/reference/vXListTickers.yml:95]: @timetraveler328

**Review by jbonzo — APPROVED** [review]
lgtm. Should get Katie's eyes

File: _specs_/rest/paths/reference/vXListTickers.yml (hunk 1 above)
Comment (timetraveler328) [reply _specs_/rest/paths/reference/vXListTickers.yml:95]: It's not possible rn

**Review by timetraveler328 — APPROVED** [review]

**Pull request merged**